src/nvofapi/nvofapi_vk_instance.cpp

```diff
@@ -27,21 +27,18 @@
 namespace dxvk {
 
     bool NvOFInstanceVk::Initialize() {
-        // For VK we cannot load winevulkan directly or we may break handle
-        // opacity.
-        m_library = LoadLibraryA("vulkan-1.dll");
-        if (!m_library) {
+        // For Vulkan we cannot load `winevulkan.dll` directly, or we may break handle opacity
+        m_vk = m_resourceFactory.CreateVulkan("vulkan-1.dll");
```

Comment:
Hmm, now that I think about it, won't this break DLFG in dxvk-remix titles? It loads winevulkan directly so it _is_ possible that vulkan-1 won't be loaded, although previous version that did `LoadLibraryA("vulkan-1.dll")` would simply load a module that forwards all its exports to winevulkan so it worked anyway.

We might now need the explicit fallback to `winevulkan.dll` here.

Reply:
Thanks. Added the fallback, including earlier missing `!m_vk` checks.